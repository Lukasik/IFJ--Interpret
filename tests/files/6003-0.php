<?php
$x = 1;
$str = "string\n";
while($str)
{
	if($x === 11)
	{
		$str = "";
	}
	else
	{
		$str = $str . $x;
		$e = put_string($str);
	}
	$x = $x + 1;
}
