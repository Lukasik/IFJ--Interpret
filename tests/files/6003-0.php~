<?php
$x = 1;
$str = "string\n";
while($str)
{
	if($x === 11)
	{
		$str = "";
	}
	else
	{
		$str = $str . $x;
		put_string($str);
	}
	$x = $x + 1;
}
