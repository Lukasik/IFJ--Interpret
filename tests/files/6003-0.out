string
1string
12string
123string
1234string
12345string
123456string
1234567string
12345678string
123456789string
12345678910